src/main/java/br/com/alura/comex/service/ProcessadorDeCsv.java

```diff
@@ -0,0 +1,55 @@
+package br.com.alura.comex.service;
+
+import br.com.alura.comex.model.Pedido;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class ProcessadorDeCsv {
+
+    public static List<Pedido> listarPedidos(String nomeArquivo) {
+        ArrayList<Pedido> pedidos = new ArrayList<>();
```

Comment:
É comum a prática de só usar uma classe concreta na instanciação de uma coleção. A partir desse momento, podemos usar um `List`.

```suggestion
       List<Pedido> pedidos = new ArrayList<>();
```